test/compiled_shaders_include/kompute_test/shaders/shadertest_logistic_regression.hpp

```diff
@@ -23,411 +23,413 @@
 
 namespace kp {
 namespace shader_data {
-static const unsigned char test_shaders_glsl_test_logistic_regression_comp_spv[] = {
```

Comment:
Also automatically generated so we shoudl skip (all the compiled shared folders)